doc/gettingstarted.md

```diff
@@ -21,32 +23,31 @@ In this guide we will get everything up and running on your local machine (on `l
 
 ## Dependencies 
 
-#### Certificates
+### CLI and Stack
+
+Get the latest packages or binaries for your operating system from the [releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on GitHub.
+
+### Certificates
+
 By default, the Stack requires a `cert.pem` and `key.pem`, in order to to serve content over TLS.
 
-+ To generate testing certificates, you can use the following command. This requires a [Go environment setup](../DEVELOPMENT.md#development-environment).
++ To generate self-signed certificates, you can use the following command. This requires a [Go environment setup](../DEVELOPMENT.md#development-environment).
 
-    ```bash
-    go run $(go env GOROOT)/src/crypto/tls/generate_cert.go -ca -host localhost && chmod 0444 ./key.pem
-    ```
-    
-+ For production certificates we recommend [LetsEncrypt](https://letsencrypt.org/getting-started/).
-    
-<a name="configuration"/>
+```bash
+go run $(go env GOROOT)/src/crypto/tls/generate_cert.go -ca -host localhost && chmod 0444 ./key.pem
```

Comment:
Note that `chmod` doesn't work on Windows

Reply:
All of the commands are marked as `bash`, and Windows isn't a first class citizen anyway (due to the Makefiles).